spec/models/factories_spec.rb

```diff
@@ -0,0 +1,11 @@
+FactoryGirl.factories.map(&:name).each do |factory_name|
```

Comment:
Could you explain what this spec does a bit?

Wasn't sure if it was to lint and validate teh factories -- or does something similar to `FactoryGirl.lint`

https://github.com/thoughtbot/factory_girl/blob/master/GETTING_STARTED.md#linting-factories